backend/uclapi/uclapi/settings.py

```diff
@@ -187,15 +187,15 @@
 
 USE_TZ = False
 
+# set it to True when developing using webpack-serve
+# so that static files are fetched from http://localhost:8080/
+WEBPACK_SERVE = True
```

Comment:
Where is this used in the code? I was going to convert it to an environment variable but I can't find the reference

Reply:
Oh, don't need it anymore now. You can remove it.